### ISSUE TITLE
iFollowTheNthLink function does not work for behat 3/php 5.5

### ISSUE THREAD

**Issue by sajjadhossain** [Behatch/contexts#130]
```
//Follows XPath element
//TODO: Fix this function

/**
 * THIS FUNCTION IS CURRENTLY DEPRECATED AND NEEDS TLC
 * Click on the nth specified link
 * Example: When I click on 1st "ul li a" link
 * Example: And I click on 6th "ul li a" link
 *
 * @When (I )follow the :index :link link
 */
public function iFollowTheNthLink($index, $link)
{
    $page = $this->getSession()->getPage();

    $links = $page->findAll('named', [
        'link', $this->getSession()->getSelectorsHandler()->xpathLiteral($link)
    ]);

    if (!isset($links[$index - 1])) {
        throw new \Exception("The $index element '$link' was not found anywhere in the page");
    }

    $links[$index - 1]->click();
}
```

The above function doesnt work. To reproduce use the step definition in any test. Error suggests not using xpathLiteral as it is deprecated.

**Comment by sajjadhossain** [comment]
error: 16384: The BehatMinkSelectorSelectorsHandler::xpathLiteral method is deprecated as of 1.7 and will be removed in 2.0. Use BehatMinkSelectorXpathEscaper::escapeLiteral instead when building Xpath or pass the unescaped value when using the named selector. in vendor/behat/mink/src/Selector/SelectorsHandler.php line 131

**Comment by sanpii** [comment]
Fixed by #221. Feel free to reopen if it’s not the case.
